docs/adapters/adapter_api.md

```diff
@@ -6,9 +6,65 @@ NOTE: This API is currently under active development and is highly likely to cha
 
 __TOC__
 
+## Overview
+
+An Adapter is the interface between the juttle runtime and a backend data source. The backend consists of a set of data points. Each data point contains a number of fields. In most cases, each data point also contains a timestamp.
```

Comment:
"The backend consists of a set of data points" -- I don't know what this means, thought a backend was like Elasticsearch or SQL?